apps/src/lib/config/genesis.rs

```diff
@@ -226,6 +226,10 @@ pub mod genesis_config {
 
     #[derive(Clone, Debug, Deserialize, Serialize)]
     pub struct ParametersConfig {
+        // Max Tendermint block size in bytes.
+        // XXX: we use i64 becauase that's what tower-abci
+        // and tendermint-rs expect
+        pub max_bytes_per_block: i64,
```

Comment:
Is this meant to be the same as `max_tx_bytes` from `PrepareProposal`? https://github.com/tendermint/tendermint/blob/v0.37.x/spec/abci/abci++_methods.md#prepareproposal

Reply:
yeah, it's the same type. btw, sorry, I didn't mean to request a review yet